src/main/java/com/flowable/flowableboot/utils/generateUmsTasks.java

```diff
@@ -0,0 +1,264 @@
+package com.flowable.flowableboot.utils;
+
+
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.ArrayList;
+import java.util.Random;
+import java.util.UUID;
+
+public class generateUmsTasks {
+    private static ArrayList<String> taskTypes = new ArrayList<>(){
+        {
+            add("Report: ");
+            add("Presentation: ");
+            add("Formal Brief: ");
+            add("Respond To Query Topic: ");
+        }
+    };
+
+    private static ArrayList<String> taskTopics = new ArrayList<>(){
+        {
+            add("Company X");
+            add("Country Y");
+            add("Topic Z");
+        }
+    };
+
+    // names from https://cybertext.wordpress.com/2007/04/30/fake-names-for-documentation/
+    private static ArrayList<String> requesterNames = new ArrayList<>(){
+        {
+            add("Simon Sais");
+            add("Paige Turner");
+            add("Chris Anthemum");
+        }
+    };
+
+    private static ArrayList<String> assigneeName = new ArrayList<>(){
+        {
+            add("Col Fays");
+            add("Rose Bush");
+            add("Hank R. Cheef");
+            add("Mike Rowe-Soft");
+            add("Penny Black");
+            add("Wiley Waites");
+        }
+    };
+
+    private static ArrayList<Integer> priority = new ArrayList<>(){
```

Comment:
I would lift this to an enum in a separate class since that will modify your DB implementation as well.

Reply:
I made an attempt at including priority, loe, and status as enums.